frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/overview/simulated-region-overview.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container>
+    <div class="popover-body">
+        <nav #nav="ngbNav" ngbNav [(activeId)]="activeNavId" class="nav-tabs">
+            <ng-container ngbNavItem="general">
+                <a ngbNavLink data-cy="transferPointPopupNamesNav">Übersicht</a>
+                <ng-template ngbNavContent>
+                    <app-general-tab [simulatedRegion]="simulatedRegion" />
+                </ng-template>
+            </ng-container>
+            <ng-container ngbNavItem="transfer">
+                <a ngbNavLink data-cy="transferPointPopupOtherTransferPointsNav"
+                    >Transfer</a
+                >
+                <ng-template ngbNavContent><app-transfer-tab /> </ng-template>
+            </ng-container>
+            <ng-container ngbNavItem="behaviors">
+                <a ngbNavLink data-cy="transferPointPopupHospitalNav"
+                    >Verhaltensweisen</a
+                >
+                <ng-template ngbNavContent><app-behavior-tab /> </ng-template>
+            </ng-container>
+        </nav>
+        <div [ngbNavOutlet]="nav" class="mt-2"></div>
+    </div>
```

Comment:
```suggestion
<ng-container>
    <nav #nav="ngbNav" ngbNav [(activeId)]="activeNavId" class="nav-tabs">
        <ng-container ngbNavItem="general">
            <a ngbNavLink data-cy="transferPointPopupNamesNav">Übersicht</a>
            <ng-template ngbNavContent>
                <app-general-tab [simulatedRegion]="simulatedRegion" />
            </ng-template>
        </ng-container>
        <ng-container ngbNavItem="transfer">
            <a ngbNavLink data-cy="transferPointPopupOtherTransferPointsNav"
                >Transfer</a
            >
            <ng-template ngbNavContent><app-transfer-tab /> </ng-template>
        </ng-container>
        <ng-container ngbNavItem="behaviors">
            <a ngbNavLink data-cy="transferPointPopupHospitalNav"
                >Verhaltensweisen</a
            >
            <ng-template ngbNavContent><app-behavior-tab /> </ng-template>
        </ng-container>
    </nav>
    <div [ngbNavOutlet]="nav" class="mt-2"></div>
</ng-container>
```

The class `popover-body` is se in the simulated region popup and doesn't need to be set a second time. By removing this, it also removes some margin to fill the available space better.